loopchain/blockchain/invoke_result.py

```diff
@@ -209,6 +209,10 @@ def height(self) -> int:
     def state_hash(self) -> Hash32:
         return self._state_hash
 
+    @property
+    def validators_hash(self) -> Hash32:
+        return self._validators_hash
+
```

Comment:
1. 
`InvokeData` has no `_validators_hash`.
Please check.

2.
It looks that this prop has no outer usage.